packages/components/tree-v2/src/composables/useTree.ts

```diff
@@ -278,10 +283,18 @@ export function useTree(
     return tree.value?.treeNodeMap.get(key)
   }
 
+  function scrollTo(key: TreeKey, align: Alignment = 'auto') {
```

Comment:
The corresponding api name of TableV2 is scrollToRow. I think it may be more standardized to rename to scrollToNode.

IMO, maybe we can expose listRef.value.scrollTo, It might be more appropriate to name it scrollTo.

WDYT

Reply:
i think both of these suggestions are good improvements that would make the API more flexible, allowing users to more easily meet their demands